src/main/content/_posts/microprofile-openapi-intro.adoc

```diff
@@ -0,0 +1,61 @@
+
+2.	Take the initial output from /openapi as a starting point to document your APIs via https://github.com/eclipse/microprofile-open-api/blob/master/spec/src/main/asciidoc/microprofile-openapi-spec.adoc#static-openapi-files[Static OpenAPI files]. It’s worth mentioning that these static files can also be written before any code, which is an approach often adopted by enterprises that want to lock-in the contract of the API. In this case, we refer to the OpenAPI document as the "source of truth", by which the client and provider must abide.
+
+3.	Use the https://github.com/eclipse/microprofile-open-api/blob/master/spec/src/main/asciidoc/microprofile-openapi-spec.adoc#programming-model[API interfaces] to provide a bootstrap (or complete) OpenAPI model tree. 
+
+4.  Additionally, a https://github.com/eclipse/microprofile-open-api/blob/master/spec/src/main/asciidoc/microprofile-openapi-spec.adoc#filter[Filter] is described which can update the OpenAPI model after it has been built from the previously described documentation mechanisms.
+
+All of these mechanisms are combined to form the valid OpenAPI v3 document from `/openapi`.  
+
+## OpenAPI UI
+
+A very important value-add from Open Liberty is the native OpenAPI UI that it ships out of the box!  This UI is built from the https://github.com/swagger-api/swagger-ui[Open Source Swagger UI] and renders the generated `/openapi` documented into a very user friendly page.  One of the best features is the ability to `Try out` each endpoint, making an invocation to the back-end straight from the UI.  
+
+The UI, available from the endpoint `/openapi/ui` can also be https://www.ibm.com/support/knowledgecenter/en/SSD28V_9.0.0/com.ibm.websphere.wlp.core.doc/ae/twlp_api_mpopenapi_custom.html[customized] beyond the default look-and-feel.  
+
+image::img/mp_ol_openapi_ui.png[MP OpenAPI UI]
+
+## Additional Information
+
+You are now ready to try out an application that has MicroProfile OpenAPI enabled!  Just head over to our  https://github.com/microservices-api/oas3-microprofile-app[sample app] and follow the instructions to build an Open Liberty docker container that with MicroProfile OpenAPI!  
+
+For more information on the MicroProfile OpenAPI specification, see the http://download.eclipse.org/microprofile/microprofile-open-api-1.0.1/microprofile-openapi-spec.html[official MicroProfile OpenAPI 1.0 release].  For further documentation the `mpOpenAPI-1.0` feature, check out the https://www.ibm.com/support/knowledgecenter/en/SSD28V_9.0.0/com.ibm.websphere.wlp.core.doc/ae/twlp_mpopenapi.html[official topic].  
```

Comment:
Suggested change:
"For further documentation the `mpOpenAPI-1.0` feature" --> "For further documentation **about** the `mpOpenAPI-1.0` feature"